### DISCUSSION TITLE
Documentation Priorities

### DISCUSSION THREAD

**Discussion by petebryan** [microsoft/msticpy#516] [General]
Feedback from the steering council and other users has been that documentation, and particularly getting started documentation could be improved for MSTICPy. 
Given the scope and scale of documentation we would like to focus on a few key areas and so would like to hear from the community on a number of things.

1. What are your top specific areas for documentation improvements, please try and be as specific as possible - whether it be a specific feature, or area (such as Installation)?
2. What format would you most like to see documentation in - formal docs, blogs, video etc?

**Comment by ianhelle** [comment]
Some thoughts for things to do:

## Getting started/intro material

Combine/align current:
- Overview/Getting started - https://msticpy.readthedocs.io/en/latest/getting_started/QuickStart.html#visualizing-data
- MSTICPy Lab and notebook

## Common Scenario Docs
Examples:
- Given an IP address, how do I find more about it?
- Set of data - extract IoCs and get context and visualize
- Encoded command lines - extract and decipher
Try to base on existing notebooks if we have them

## Improve the mp help

## Ease of use for non-pivot, non-pd accessor functionality
How do we make things like IoC extract easier to use?


## Deeper training
- High level (30min) sections on data, enrich, analyze, visualize
- plus a couple of housekeeping areas like install, configure
- Expanded versions (60m) for each of these 

## Tutorial content
- Can go in Getting started or new Tutorial section
- Same content as 30 min sessions - with linked notebooks
